include/PruManager.h

```diff
@@ -35,15 +41,50 @@ class PruManager
 };
 
 #if ENABLE_PRU_RPROC == 1
+/**
+ * \brief Support for interaction with PRU via
+ * Remote Proc + Mmap
+ * \nosubgrouping
+ */
 class PruManagerRprocMmap : public PruManager
 {
-// use rproc for start/stop and mmap for memory sharing
 public:
+	/**
```

Comment:
Okay then I am leaving the base class' pruNum parameter empty as it is child class dependent.

Reply:
I'd rather have it in the base class instead, so that children classes know what the specs are. Then if they have exceptions, they should document it appropriately.

Reply:
Okay, added in 7c0e7b97